frontend/src/component/insights/componentsChart/CreationArchiveChart/CreationArchiveChart.tsx

```diff
@@ -203,6 +208,11 @@ export const CreationArchiveChart: FC<ICreationArchiveChartProps> = ({
                 options={options}
                 height={100}
                 width={250}
+                plugins={[
+                    customHighlightPlugin({
+                        maxHighlightOpacity: 0.24,
```

Comment:
The default highlight opacity was much too subtle on this graph, so we had to up the max opacity. Here's what it looks like with 0.12:

<img width="1112" height="782" alt="Image" src="https://github.com/user-attachments/assets/f7b8048b-0175-46e3-bc28-c47ec811ee1c" />